src/DittoMachine.sol

```diff
@@ -309,7 +310,7 @@ contract DittoMachine is ERC721, ERC721TokenReceiver, ERC1155TokenReceiver {
 
             // reduce heat relative to amount of time elapsed by auction
             if (cloneIdToShape[cloneId].term > block.timestamp) {
-                uint256 termLength = (BASE_TERM-1) + heat**2;
+                uint256 termLength = (BASE_TERM) + TimeCurve.calc(heat);
```

Comment:
we need to document the behaviour of termLength and heat calculation comprehensively. Either here or in the mdbook.

Reply:
totally agree, added some more comments here, but we can definitely go into more depth in the documentation.